config/default/kustomization.yaml

```diff
@@ -135,10 +144,70 @@ patches:
 #          kind: Certificate
 #          group: cert-manager.io
 #          version: v1
+#          name: serving-cert
+#        fieldPaths:
+#          - .spec.dnsNames.0
+#          - .spec.dnsNames.1
+#        options:
+#          delimiter: '.'
+#          index: 1
+#          create: true
+#  - source: # Add cert-manager annotation to the webhook Service
+#      kind: Service
+#      version: v1
+#      name: controller-manager-metrics-service
+#      fieldPath: .metadata.name # namespace of the service
+#    targets:
+#      - select:
+#          kind: Certificate
+#          group: cert-manager.io
+#          version: v1
+#          name: metrics-certs
```

Comment:
and in some places we use metrics-certs

Reply:
kustomize prepends `lws` https://github.com/kubernetes-sigs/lws/blob/66bcbc7f000dd65e66719aba7f5f9375bf7a00b5/config/default/kustomization.yaml#L9 to the resources and this will be formed to `lws-metrics-certs`. If there is any other value, it wouldn't be correct.

Reply:
Thank you for warning me. Helm chart should be `lws-metrics-cert` and kustomize `metrics-cert` (not `metrics-certs`). I've updated the code.